src/run/docker-compose.ts

```diff
@@ -200,7 +202,7 @@ export class DockerCompose {
     */
     private get getAwsCredentials (): Record<string, string> {
         try {
-            const output = execSync("aws configure export-credentials --format env", { encoding: "utf-8" });
+            const output = execSync(`aws configure export-credentials --profile ${AWS_PROFILE} --format env`, { encoding: "utf-8" });
```

Comment:
So what happens in this method if AWS_PROFILE is undefined? Should it detect that and fail with a useful error message?

Reply:
I think it would be best not adding `--profile` if AWS_PROFILE is undefined just incase someone has a profile `undefined` set and cause all sorts of catastrophe

Reply:
Or error as AWS_PROFILE is required to make all this work (this maintains the current behaviour)

Reply:
Yup.